test-requirements.txt

```diff
@@ -2,7 +2,8 @@ mock
 pur
 # pytest==3.1.0 issues with pytest-verbose-parametrize
 pytest==3.0.7
-pytest-cov==2.6.0
+pytest-cov<2.6.1; python_version < "3.6"
```

Comment:
the changed requirement is for [pytest>=3.6](https://github.com/pytest-dev/pytest-cov/compare/v2.6.0...master#diff-2eeaed663bd0d25b7e608891384b7298R124) and not python 3.6

Reply:
So it should be pinned with markers too